### PR TITLE
Rename from `node-sdk` to `js-sdk`

### PR DESCRIPTION
**Pull request by colinking**
I renamed this repo to `js-sdk` and this PR updates references from `node-sdk` to `js-sdk`. This change indicates that this SDK can be used in various JS runtimes, not just Node, e.g. Views (where it runs client-side) and Workflows (which uses the V8 runtime).

Resolves TSK-214

### CODE REVIEW

**Comment by colinking** [comment]
Current dependencies on/for this PR:
* main
  * **PR #121** <a href="https://app.graphite.dev/github/pr/airplanedev/node-sdk/121" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/airplanedev/node-sdk/121?utm_source=stack-comment).

File: packages/airplane/internal/config.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ export type TaskConfig<TParams extends Params> = {
   runtime?: "standard" | "workflow";
   resources?: Record<string, string> | string[];
   schedules?: Record<string, Schedule<TParams>>;
-  nodeVersion?: "12" | "14" | "15" | "16" | "18";
+  nodeVersion?: "14" | "16" | "18";
```
Comment (colinking) [inline packages/airplane/internal/config.ts:55]: cc @ameyakhare @calufornia I noticed we accepted Node 12 and 15. Was that intentional? I've removed it here.

File: packages/airplane/internal/api/fetcher.ts [hunk 2]
```diff
@@ -89,7 +89,7 @@ export class Fetcher {
     url.pathname = path;
     url.search = params ? querystring.stringify(params) : "";
     const headers: HeadersInit = {
-      "X-Airplane-Client-Kind": "sdk/node",
+      "X-Airplane-Client-Kind": "sdk/js",
```
Comment (colinking) [inline packages/airplane/internal/api/fetcher.ts:92]: I'll open a PR on `airport` to accept the new client kind.

Comment (colinking) [reply packages/airplane/internal/api/fetcher.ts:92]: https://github.com/airplanedev/airport/pull/3128

**Review by calufornia — APPROVED** [review]
🚀

**Review by ameyakhare — APPROVED** [review]

**Pull request merged**